samples/recognize.v1p1beta1.js

```diff
@@ -141,7 +141,7 @@ function syncRecognizeWithAutoPunctuation(
   // [START speech_transcribe_file_with_auto_punctuation]
   // Imports the Google Cloud client library
   const fs = require('fs');
-  const speech = require('@google-cloud/speech');
+  const speech = require('@google-cloud/speech').v1p1beta1;
```

Comment:
Even though we want to provide examples for beta endpoints, I don't really like that because it makes disabling beta endpoints much harder. Can we at least have a comment here that this code sample uses beta functionality that will eventually be disabled?

Reply:
Fair point. I will add the comment now.

Reply:
First of all, I'm not the one who should make the decision here - it's @jmdobry, @crwilcox, and @jonparrott. With that disclaimer, my understanding is that users copy the samples code to their applications and we cannot be sure that every time the user copies `.v1p1beta1` they do create a task in their bugtracker to make sure to migrate from the beta endpoint whenever the beta gets disabled. I want to send a clear message to all the beta users that they are _beta users_, and just having `.v1p1beta1` in the `require` line is definitely not enough.

Reply:
Also note that per our policy, disabling beta in the code when the beta service goes down is just a semver minor change (so most likely users' code using beta endpoint will just break as most users don't pin their dependencies but use `^` instead).

Reply:
> We cannot be sure that every time the user copies .v1p1beta1 they do create a task in their bugtracker to make sure to migrate from the beta endpoint whenever the beta gets disabled - @alexander-fenster 

Fair.

> Perhaps the fact these samples reside in recognize.**v1p1beta1**.js and not in recognize.js, communicates this to the users already - @happyhuman 

Also Fair.

How about we use something like this as standard messaging for using non-default API endpoints in dynamic language client libraries?

`[Default Endpoint]`

```js
// Imports the Google Cloud client library
const speech = require('@google-cloud/speech');
```

`[Explicit Beta Endpoint]`

```js
// Imports the Google Cloud client library for Beta API
/**
 * TODO(developer): Update client library import to use new
 * version of API when desired features become available
 */
const speech = require('@google-cloud/speech').v1p1beta1;
```

Here is an example of how we show Beta client libraries on the website:
https://cloud.google.com/vision/docs/beta-libraries

Reply:
So, are we adding the TODO for the new code in this PR only? Or for all occurrences of `require('@google-cloud/speech').v1p1beta1;` anywhere in this repo (which there are many of them)?

Reply:
Let's add to the new code in scope of this PR, and make a separate PR that addresses this problem for other samples. I don't want to mix things together, but I do want to send a message to all beta users.